grr/server/grr_response_server/bin/grrafana.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+# Lint as: python3
+from absl import app, flags
+import json
+import os
+from typing import Any, cast, Dict, List, Text, Tuple
+
+from fleetspeak.src.server.proto.fleetspeak_server import resource_pb2
+
+from grr_response_core import config
+from grr_response_core.config import server as config_server
+from grr_response_core.config import contexts
+from grr_response_server import fleetspeak_connector
+from grr_response_server import fleetspeak_utils
+from grr_response_server import server_startup
+
+from werkzeug import exceptions as werkzeug_exceptions
+from werkzeug import routing as werkzeug_routing
+from werkzeug import serving as werkzeug_serving
+from werkzeug import wrappers as werkzeug_wrappers
+from werkzeug.wrappers import json as werkzeug_wrappers_json
+
+
+AVAILABLE_METRICS = [
+    "mean_user_cpu_rate",
+    "max_user_cpu_rate",
+    "mean_system_cpu_rate",
+    "max_system_cpu_rate",
+    "mean_resident_memory_mib",
+    "max_resident_memory_mib",
+]
+
+JSON_MIME_TYPE = "application/json"
+
+flags.DEFINE_bool(
+    "version",
+    default=False,
+    allow_override=True,
+    help="Print the GRR console version number and exit immediately.")
+
+
+class JSONRequest(werkzeug_wrappers_json.JSONMixin, werkzeug_wrappers.Request):
+  pass
+
+
+class JSONResponse(werkzeug_wrappers_json.JSONMixin, werkzeug_wrappers.Response):
+  pass
+
+
+class Grrafana(object):
+  """GRRafana HTTP server instance.
+  
+  A full description of all endpoints implemented within this HTTP
+  server can be found in https://github.com/simPod/grafana-json-datasource#api."""
+
+  def __init__(self) -> None:
+    """Constructor."""
```

Comment:
"Constructor." violates [pystyle](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) and contains no information. Please avoid non-descriptive docstrings. I think our linter would complain about missing docstrings, so use something like `"""Initializes a new GRRafana HTTP server instance."""`